README.md

```diff
@@ -10,6 +10,7 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 - `odf get`:
   - `recovery-profile`: Get the recovery profile value.
   - `health`: Check health of the cluster and common configuration issues.
+  - `dr-health [ceph status args]`: Print the ceph status of a peer cluster in a mirroring-enabled environment thereby validating connectivity between ceph clusters. Ceph status args can be optionally passed, such as to change the log level: --debug-ms 1.
```

Comment:
The "ceph status args" are specifically passed to the `ceph status` command and nothing else?

Reply:
yes, correct